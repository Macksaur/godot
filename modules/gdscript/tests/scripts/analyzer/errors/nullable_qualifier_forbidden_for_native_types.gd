func test():
    var input: InputMap? = InputMap # Should error
