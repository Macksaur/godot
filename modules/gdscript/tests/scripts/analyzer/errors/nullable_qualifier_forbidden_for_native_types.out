GDTEST_ANALYZER_ERROR
Cannot assign a value of type GDScriptNativeClass to variable "input" with specified type InputMap?.
